packages/sdk/src/methods/token.test.ts

```diff
@@ -17,31 +24,30 @@ describe('Test token', () => {
   it('should get token infos', async () => {
     const name = randomName();
     const resolverAddress = Address.fromRandom().toB256();
+    const domainHash = sha256(domainToBytes(name));
 
     await register({
       domain: name,
       account: wallet,
       resolver: resolverAddress,
     });
 
-    const {
-      image,
-      subId,
-      symbol,
-      contractId,
-      name: tokenName,
-    } = await tokenInfo(name);
-
-    expect(tokenName).toBe('Bako ID');
-    expect(image).toBe(`https://assets.bako.id/${name}`);
-    expect(symbol).toBe('BNFT');
-    expect(subId).toBe(sha256(domainToBytes(name)));
-    expect(contractId).toBe(config.REGISTRY_CONTRACT_ID);
+    const token = await tokenInfo(name, { provider });
+
+    expect(token?.name).toBe('Bako ID');
+    expect(token?.image).toBe(`https://assets.bako.id/${name}`);
+    expect(token?.symbol).toBe('BNFT');
+    expect(token?.assetId).toBe(
+      getMintedAssetId(config.REGISTRY_CONTRACT_ID, domainHash),
+    );
+    expect(token?.subId).toBe(domainHash);
+    expect(token?.contractId).toBe(config.REGISTRY_CONTRACT_ID);
   });
 
   it('should undefined values on token not minted', async () => {
     const name = randomName();
     const tokenResult = await tokenInfo(name, { provider });
+
     expect(tokenResult).toBeUndefined();
   });
```

Comment:
this test is failing because the validation of whether the token exists was removed

https://github.com/infinitybase/bako-id/blob/fd4ba0ea4e9566406c58fd80d0698d4a3d7495b2/packages/sdk/src/methods/token.ts#L47-L49

Reply:
I added again in the previous [commit](https://github.com/infinitybase/bako-id/pull/40/commits/27c5b3cc00c7db5fa89c155fa4bd2f3d921220c8)